packages/dapp-svelte-wallet/react-ui/src/views/Issuers.jsx

```diff
@@ -1,7 +1,79 @@
+import { useState } from 'react';
 import { Typography } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import Add from '@material-ui/icons/Add';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Card from '../components/Card';
+import CardItem from '../components/CardItem';
+import MakePurse from '../components/MakePurse';
+import { withApplicationContext } from '../contexts/Application';
 
-const Issuers = () => {
-  return <Typography variant="h1">Issuers</Typography>;
+import './Issuers.scss';
+
+// Exported for testing only.
+export const IssuersInternalDoNotImport = ({
```

Comment:
Can you rename these kinds of things to:
```suggestion
export const IssuersWithoutContext = ({
```

Reply:
Ah yea that's a bit better, done